automata/fa/nfa.py

```diff
@@ -123,7 +123,7 @@ def _from_symbol(cls, symbol, input_symbols=None):
     @classmethod
     def from_regex(cls, regex):
         """Initialize this NFA as one equivalent to the given regular expression"""
-        input_symbols = set(regex) - {'*', '|', '(', ')', '?'}
+        input_symbols = set(regex) - {'*', '|', '(', ')', '?', ' ', '\t'}
```

Comment:
@eliotwrobson What is the purpose of the extra regex symbols?

Reply:
Allows regex to ignore whitespace. The lexer is already configured to skip whitespace by default, without this change regexes will have whitespace passed in as input symbols.

Reply:
@eliotwrobson Is whitespace not significant in a automata theory regex (like it is in Perl regex)?

Reply:
The way it's configured now it's just ignored. You can't use whitespace as implicit parentheses. 

This behavior can actually be changed but that's a little beyond the scope of this PR

Reply:
@eliotwrobson Makes sense. This looks good, then.